apps/climatemappedafrica/src/components/Section/index.js

```diff
@@ -1,46 +1,30 @@
-import { Section as CuiSection } from "@commons-ui/legacy";
-import makeStyles from "@mui/styles/makeStyles";
-import clsx from "clsx";
+import { Section as CuiSection } from "@commons-ui/core";
 import PropTypes from "prop-types";
 import React from "react";
 
-const useStyles = makeStyles(({ breakpoints, typography, widths }) => ({
-  root: {
-    padding: `0 ${typography.pxToRem(20)}`,
-  },
-  /* Styles applied to the root element if `fixed={true}`. */
-  fixed: Object.keys(widths.values).reduce((acc, breakpoint) => {
-    const value = widths.values[breakpoint];
-    if (value !== 0) {
-      acc[breakpoints.up(breakpoint)] = {
-        padding: 0,
-        width: value,
-      };
-    }
-    return acc;
-  }, {}),
-}));
-
-function Section({ className, fixed = true, ...props }) {
-  const classes = useStyles(props);
-
+function Section({ fixed = true, sx, ...props }) {
```

Comment:
Do we still need custom `Section`? Ideally the `commons/ui/Section` component should be flexible/reusable enough.

Reply:
@kilemensi yes. We have to give each `Section` a fixed width. Rather than do so at each place it is used, having a custom one is much better and cleaner.

Reply:
🤔 ... I just glanced at the code @kelvinkipruto and could be wrong but isn't the [code](https://github.com/CodeForAfrica/ui/blob/9c25d295feac00d325473af7d6d9b4347bef879e/apps/climatemappedafrica/src/components/Section/index.js#L11-L20) in this PR exactly like the one in [`commons-ui/core/Section`](https://github.com/CodeForAfrica/ui/blob/41f5cdc0cb3e7849457bdfc297b47a2c91493ee3/packages/commons-ui-core/src/Section/Section.js#L16-L28) already? Of course, `theme.widths` was renamed to `theme.contentWidths`.

Reply:
@kilemensi The main issue is the width. If we use the `commons-ui/core` `section`, it is wider than the original.
![2025-01-13_11-19](https://github.com/user-attachments/assets/348707b2-e93f-4626-b58f-0c3abc254546)
![after](https://github.com/user-attachments/assets/7bf94863-d3a5-4500-8a31-4179722928f1)

Reply:
Cool @kelvinkipruto ... to confirm did you update the [theme](https://github.com/CodeForAfrica/ui/blob/41f5cdc0cb3e7849457bdfc297b47a2c91493ee3/apps/climatemappedafrica/src/theme/index.js#L133-L138) to:
```js
  contentWidths: {
    values: {
      md: 608, // 0, 80, 0, 80 margin
      lg: 1160, // 0, 140, 0, 140 margin
    },
  },
```

Reply:
This works. Thanks @kilemensi.